src/pathEqual.ts

```diff
@@ -1,17 +1,23 @@
-export function pathEqual(actual: string, expected: string) {
-  if (actual === expected) return true
```

Comment:
I would keep this here because the normalization takes time. If we can return early without doing it, it is better.

Reply:
That's ok .